addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -213,7 +213,7 @@
                                             </t>
                                         </t>
                                         <a t-if="channel.can_upload"
-                                            class="o_wslides_js_channel_tag_add border badge text-bg-light fw-normal py-1 m-1 o_not_editable"
```

Comment:
Somewhere in slides, not especially on this line, but:
before
![image](https://github.com/odoo/odoo/assets/30048408/bb5db482-0785-4f97-b9c8-8ac84273b79c)
after
![image](https://github.com/odoo/odoo/assets/30048408/94fffd82-20d5-4e76-972e-354d41dbb10a)

Can you make something to add back the space between the rows?

Reply:
Hello, this is should be fixed now 👍